content/chapitres/cli.adoc

```diff
@@ -0,0 +1,209 @@
+[{invert}]
+= Guide de survie en ligne de commande
+
+Remise à niveau / Rappels
+
+[{invert}]
+== CLI
+
+- 🇬🇧 CLI == "Command Line Interface"
+- 🇫🇷 "Interface de Ligne de Commande"
+
+== REPL
+
+Pour les théoriciens et curieux :
+
+- 🇬🇧 REPL == "Read–eval–print loop"
+
+https://en.wikipedia.org/wiki/Read%E2%80%93eval%E2%80%93print_loop[REPL sur Wikipedia,window="_blank"]
+
+== Anatomie d'une commande
+
+[source,bash]
+----
+ls --color=always -l /bin
+----
+
+* Séparateur : l'espace
+* Premier élément (`ls`) : c'est la commande
+* Les éléments commençant par un tiret `-` sont des "options" et/ou drapeaux ("flags")
+** "Option" == "Optionnel"
+* Les autres éléments sont des arguments  (`/bin`)
+** Nécessaire (par opposition)
+
+== Manuel des commande
+
+Afficher le manuel de `<commande>` :
+
+[source,bash]
+man <commande>
+
+* Navigation avec les flèches haut et bas
+* Tapez `/` puis une chaîne de texte pour chercher
+** Touche `n` pour sauter d’occurrence en occurrence
+* Touche `q` pour quitter
+
+🎓 Essayez avec `ls`, chercher le mot `color`
+
+[.small]
+Google c'est pratique aussi hein !
+
+== Raccourcis
+
+Dans un terminal Unix/Linux/WSL :
+
+* `CTRL + C` : Annuler le process ou prompt en cours
+* `CTRL + L` : Nettoyer le terminal
+* `CTRL + A` : Positionner le curseur au début de la ligne
+* `CTRL + E` : Positionner le curseur à la fin de la ligne
+* `CTRL + R` : Rechercher dans l'historique de commandes
+
+[.small]
+🎓 Essayez-les !
+
+== Commandes de base 1/2
+
+* `pwd` : Afficher le répertoire courant
+** 🎓 Option `-P` ?
+* `ls` : Lister le contenu du répertoire courant
+** 🎓 Options `-a` et `-l` ?
+* `cd` : Changer de répertoire
+** 🎓 Sans argument : que se passe t'il ?
+* `cat` : Afficher le contenu d'un fichier
+** 🎓 Essayez avec plusieurs arguments
+* `mkdir` : créer un répertoire
+** 🎓 Option `-p` ?
+
+== Commandes de base 2/2
+
+* `echo` : Afficher un (des) message(s)
+* `rm` : Supprimer un fichier ou dossier
+* `touch` : Créer un fichier
+* `grep` : Chercher un motif de texte
+
+== Arborescence de fichiers
```

Comment:
Suggestion: How about mentioning a bit of theory regarding how the file system is represented on Linux?

In other words, make 100% sure that they catch that the filesyststem is a tree, and then giving them the necessary tools to navigate this tree?

Reply:
Done, split the slide in 2. WDYT?